Applications/shapeworks/Commands.cpp

```diff
@@ -263,6 +287,34 @@ int SmoothMesh::execute(const optparse::Values &options, SharedCommandData &shar
   return sharedData.mesh.smooth(/*maxRMSErr, numIter*/);
 }
 
+///////////////////////////////////////////////////////////////////////////////
+// Coverage
+///////////////////////////////////////////////////////////////////////////////
+void Coverage::buildParser()
+{
+  const std::string prog = "coverage";
+  const std::string desc = "coverage between two meshes";
+  parser.prog(prog).description(desc);
+  parser.add_option("--second_mesh").action("store").type("string").set_default("").help("Second mesh to apply coverage.");
```

Comment:
What is the "second_mesh"? Is it a target to which "coverage" is applied? Is it therefore the result of the command? There just isn't enough information here for the command to be understood. I'm creating an issue about this (add detailed documentation shown only for <subcommand> --help), but for now maybe better to add a bit more description.